{
  "uid" : "f702f3d34dac6e76",
  "name" : "Проверка текста заголовка 'Восстановление пароля'",
  "fullName" : "tests.test_password_recovery.test_password_recovery.TestPasswordRecoveryPage#test_get_title_recovery",
  "historyId" : "50081c66d484804a855ccb4ea0b15c95",
  "time" : {
    "start" : 1714443608522,
    "stop" : 1714443608522,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x559445901e33 <unknown>\n#1 0x5594455f1567 <unknown>\n#2 0x5594455e9879 <unknown>\n#3 0x5594455d9852 <unknown>\n#4 0x5594455dae44 <unknown>\n#5 0x5594455d9bbe <unknown>\n#6 0x5594455d92e0 <unknown>\n#7 0x5594455d9181 <unknown>\n#8 0x5594455d70ec <unknown>\n#9 0x5594455d760a <unknown>\n#10 0x5594455f3eef <unknown>\n#11 0x55944567d695 <unknown>\n#12 0x55944565e362 <unknown>\n#13 0x55944567cae2 <unknown>\n#14 0x55944565e103 <unknown>\n#15 0x55944562f1ba <unknown>\n#16 0x55944562fb8e <unknown>\n#17 0x5594458c659b <unknown>\n#18 0x5594458ca34b <unknown>\n#19 0x5594458b2681 <unknown>\n#20 0x5594458caeb2 <unknown>\n#21 0x5594458976af <unknown>\n#22 0x5594458f1428 <unknown>\n#23 0x5594458f1600 <unknown>\n#24 0x559445900f84 <unknown>\n#25 0x7fa529094ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"25547922c8b256f55e46d42901bf957f\")>\n\n    @pytest.fixture(scope='function')\n    def recovery_page_open(driver):\n        recovery_page = PasswordRecoveryPage(driver)\n>       driver.get(Constant.PASSWORD_RECOVERY)\n\ntests/test_password_recovery/conftest.py:9: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:356: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f17c7c0a540>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5594458f1600 \\\\u003Cunknown>\\\\n#24 0x559445900f84 \\\\u003Cunknown>\\\\n#25 0x7fa529094ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome-headless-shell=124.0.6367.60)\nE       Stacktrace:\nE       #0 0x559445901e33 <unknown>\nE       #1 0x5594455f1567 <unknown>\nE       #2 0x5594455e9879 <unknown>\nE       #3 0x5594455d9852 <unknown>\nE       #4 0x5594455dae44 <unknown>\nE       #5 0x5594455d9bbe <unknown>\nE       #6 0x5594455d92e0 <unknown>\nE       #7 0x5594455d9181 <unknown>\nE       #8 0x5594455d70ec <unknown>\nE       #9 0x5594455d760a <unknown>\nE       #10 0x5594455f3eef <unknown>\nE       #11 0x55944567d695 <unknown>\nE       #12 0x55944565e362 <unknown>\nE       #13 0x55944567cae2 <unknown>\nE       #14 0x55944565e103 <unknown>\nE       #15 0x55944562f1ba <unknown>\nE       #16 0x55944562fb8e <unknown>\nE       #17 0x5594458c659b <unknown>\nE       #18 0x5594458ca34b <unknown>\nE       #19 0x5594458b2681 <unknown>\nE       #20 0x5594458caeb2 <unknown>\nE       #21 0x5594458976af <unknown>\nE       #22 0x5594458f1428 <unknown>\nE       #23 0x5594458f1600 <unknown>\nE       #24 0x559445900f84 <unknown>\nE       #25 0x7fa529094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1714443608523,
      "stop" : 1714443608720,
      "duration" : 197
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1714443483373,
      "stop" : 1714443483373,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1714443483355,
      "stop" : 1714443483372,
      "duration" : 17
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "recovery_page_open",
    "time" : {
      "start" : 1714443608720,
      "stop" : 1714443608938,
      "duration" : 218
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x559445901e33 <unknown>\n#1 0x5594455f1567 <unknown>\n#2 0x5594455e9879 <unknown>\n#3 0x5594455d9852 <unknown>\n#4 0x5594455dae44 <unknown>\n#5 0x5594455d9bbe <unknown>\n#6 0x5594455d92e0 <unknown>\n#7 0x5594455d9181 <unknown>\n#8 0x5594455d70ec <unknown>\n#9 0x5594455d760a <unknown>\n#10 0x5594455f3eef <unknown>\n#11 0x55944567d695 <unknown>\n#12 0x55944565e362 <unknown>\n#13 0x55944567cae2 <unknown>\n#14 0x55944565e103 <unknown>\n#15 0x55944562f1ba <unknown>\n#16 0x55944562fb8e <unknown>\n#17 0x5594458c659b <unknown>\n#18 0x5594458ca34b <unknown>\n#19 0x5594458b2681 <unknown>\n#20 0x5594458caeb2 <unknown>\n#21 0x5594458976af <unknown>\n#22 0x5594458f1428 <unknown>\n#23 0x5594458f1600 <unknown>\n#24 0x559445900f84 <unknown>\n#25 0x7fa529094ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/tests/test_password_recovery/conftest.py\", line 9, in recovery_page_open\n    driver.get(Constant.PASSWORD_RECOVERY)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1714443608969,
      "stop" : 1714443609027,
      "duration" : 58
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Тестирование страницы 'Восстановление пароля'"
  }, {
    "name" : "tag",
    "value" : "regress"
  }, {
    "name" : "parentSuite",
    "value" : "tests.test_password_recovery"
  }, {
    "name" : "suite",
    "value" : "test_password_recovery"
  }, {
    "name" : "subSuite",
    "value" : "TestPasswordRecoveryPage"
  }, {
    "name" : "host",
    "value" : "fv-az1385-609"
  }, {
    "name" : "thread",
    "value" : "2312-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_password_recovery.test_password_recovery"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "browser",
    "value" : "'chrome'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 11,
        "unknown" : 0,
        "total" : 14
      },
      "items" : [ {
        "uid" : "4f8e8be5f7487971",
        "reportUrl" : "https://Nat754.github.io/PulseWave/9//#testresult/4f8e8be5f7487971",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x561fd10c8e33 <unknown>\n#1 0x561fd0db8567 <unknown>\n#2 0x561fd0db0879 <unknown>\n#3 0x561fd0da0852 <unknown>\n#4 0x561fd0da1e44 <unknown>\n#5 0x561fd0da0bbe <unknown>\n#6 0x561fd0da02e0 <unknown>\n#7 0x561fd0da0181 <unknown>\n#8 0x561fd0d9e0ec <unknown>\n#9 0x561fd0d9e60a <unknown>\n#10 0x561fd0dbaeef <unknown>\n#11 0x561fd0e44695 <unknown>\n#12 0x561fd0e25362 <unknown>\n#13 0x561fd0e43ae2 <unknown>\n#14 0x561fd0e25103 <unknown>\n#15 0x561fd0df61ba <unknown>\n#16 0x561fd0df6b8e <unknown>\n#17 0x561fd108d59b <unknown>\n#18 0x561fd109134b <unknown>\n#19 0x561fd1079681 <unknown>\n#20 0x561fd1091eb2 <unknown>\n#21 0x561fd105e6af <unknown>\n#22 0x561fd10b8428 <unknown>\n#23 0x561fd10b8600 <unknown>\n#24 0x561fd10c7f84 <unknown>\n#25 0x7ff6e8094ac3 <unknown>",
        "time" : {
          "start" : 1714357260520,
          "stop" : 1714357260520,
          "duration" : 0
        }
      }, {
        "uid" : "9654cda6ccfffbb3",
        "reportUrl" : "https://Nat754.github.io/PulseWave/8//#testresult/9654cda6ccfffbb3",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x559ee3907e33 <unknown>\n#1 0x559ee35f7567 <unknown>\n#2 0x559ee364227d <unknown>\n#3 0x559ee3642331 <unknown>\n#4 0x559ee3685624 <unknown>\n#5 0x559ee366438d <unknown>\n#6 0x559ee3682ae2 <unknown>\n#7 0x559ee3664103 <unknown>\n#8 0x559ee36351ba <unknown>\n#9 0x559ee3635b8e <unknown>\n#10 0x559ee38cc59b <unknown>\n#11 0x559ee38d034b <unknown>\n#12 0x559ee38b8681 <unknown>\n#13 0x559ee38d0eb2 <unknown>\n#14 0x559ee389d6af <unknown>\n#15 0x559ee38f7428 <unknown>\n#16 0x559ee38f7600 <unknown>\n#17 0x559ee3906f84 <unknown>\n#18 0x7ff274694ac3 <unknown>",
        "time" : {
          "start" : 1714273789537,
          "stop" : 1714273789537,
          "duration" : 0
        }
      }, {
        "uid" : "725db891e8e6ba0a",
        "reportUrl" : "https://Nat754.github.io/PulseWave/7//#testresult/725db891e8e6ba0a",
        "status" : "passed",
        "time" : {
          "start" : 1714185194100,
          "stop" : 1714185194137,
          "duration" : 37
        }
      }, {
        "uid" : "3fed0f02c211c16b",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/3fed0f02c211c16b",
        "status" : "passed",
        "time" : {
          "start" : 1714148623530,
          "stop" : 1714148623569,
          "duration" : 39
        }
      }, {
        "uid" : "d82c37f676fbc3a8",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/d82c37f676fbc3a8",
        "status" : "passed",
        "time" : {
          "start" : 1714098673533,
          "stop" : 1714098673570,
          "duration" : 37
        }
      }, {
        "uid" : "dee6dda534c02916",
        "reportUrl" : "https://Nat754.github.io/PulseWave/5//#testresult/dee6dda534c02916",
        "status" : "passed",
        "time" : {
          "start" : 1714014530862,
          "stop" : 1714014530899,
          "duration" : 37
        }
      }, {
        "uid" : "d9cea01e25106f70",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/d9cea01e25106f70",
        "status" : "passed",
        "time" : {
          "start" : 1713946037505,
          "stop" : 1713946037542,
          "duration" : 37
        }
      }, {
        "uid" : "f6922488e6c18e18",
        "reportUrl" : "https://Nat754.github.io/PulseWave/4//#testresult/f6922488e6c18e18",
        "status" : "passed",
        "time" : {
          "start" : 1713925828116,
          "stop" : 1713925828152,
          "duration" : 36
        }
      }, {
        "uid" : "8d28b14a6f3ebbd",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/8d28b14a6f3ebbd",
        "status" : "passed",
        "time" : {
          "start" : 1713901701882,
          "stop" : 1713901701920,
          "duration" : 38
        }
      }, {
        "uid" : "b4ad02829fadf6ba",
        "reportUrl" : "https://Nat754.github.io/PulseWave/210//#testresult/b4ad02829fadf6ba",
        "status" : "passed",
        "time" : {
          "start" : 1713885931821,
          "stop" : 1713885931858,
          "duration" : 37
        }
      }, {
        "uid" : "a3225d68bd8e497b",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/a3225d68bd8e497b",
        "status" : "passed",
        "time" : {
          "start" : 1713883819078,
          "stop" : 1713883819115,
          "duration" : 37
        }
      }, {
        "uid" : "dc0d1b62236f8d8a",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/dc0d1b62236f8d8a",
        "status" : "passed",
        "time" : {
          "start" : 1713882349771,
          "stop" : 1713882349809,
          "duration" : 38
        }
      }, {
        "uid" : "ba6c9c629a281bf0",
        "reportUrl" : "https://Nat754.github.io/PulseWave/208//#testresult/ba6c9c629a281bf0",
        "status" : "passed",
        "time" : {
          "start" : 1713879529198,
          "stop" : 1713879529237,
          "duration" : 39
        }
      } ]
    },
    "tags" : [ "regress" ]
  },
  "source" : "f702f3d34dac6e76.json",
  "parameterValues" : [ "'chrome'" ]
}